### DISCUSSION TITLE
Daily 08/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#230] [Daily Sprint]
- Qué hice ayer?
- Qué haré hoy
- Divicultades o Impedimentos

**Comment by rpgrca** [comment]
Hoy amanecí engripado así que no creo que pueda grabar el video sobre los servicios con voz ronca. Tal vez lo haga mañana. Voy a leer lo que se pide para este sprint que empezó y tal vez agregar algunas issues.

**Comment by NataliaAlvarezIspc** [comment]
El fin de semana terminamos con el sprint y tuvimos nuestra reunión de revisión y planificación.
Hoy voy a estar creando las issue para el sprint 2. También voy a estar mirando videos sobre los temas del sprint que viene.

**Comment by BraianTroncoso** [comment]
Hoy estuve haciendo tareas puntuales en casa, y tengo clases de Python y Js sobre OOP.
Hoy solamente me planteo ver eso, si llego veo algo sobre servicios, no sé.
